Ports Random Tip of the Round System to Round Start. (#1488)
UristMcStation/UristMcStation: config/tips.txt

Reviewer: This either needs to be removed before merge, or replaced with actual tips.

Some exo-planets may have flammable atmospheres, laser weaponry, smoking and gun-fire may set areas, or the entire planet on fire.
Long-Range Holocommunicators allow you to talk to other vessels in the sector, provided someone is around on the other end to reply.
Short-wave radios allow you to communicate with each-other in areas without telecommunications.
Abandoned and derelict ruins may hold ancient treasures, or horrors.
No two alien anomalies will have the same activation state, use the anomaly computer to analyze and outline what you may need to do to activate them.
It is considered poor taste to decapitate someone and keep their neural lace, always give people a chance to come back into the round in some way.
Straightjackets and Electropacks should never be used without admin approval, as they are impossible to get out of.
Stowaways have a chance to be spawnable after the round starts, however as a Stowaway you won't be defined on the crew manifest or have any ID.
Most ship AI will start with the manifest lawset, making it follow orders of crew on the ship manifest, as defined by the Captain.

> Ionispheric anomalies may cause AI's to gain new laws, making them act in unusual or fun manners.

Reviewer: ```suggestion
Ionospheric anomalies may cause AIs to gain new laws, making them act in unusual or fun manners.
```

> You can use Intellicards to card AI's into a portable state, allowing them to be repaired and have their lawsets visible.

Reviewer: ```suggestion
You can use Intellicards to card AIs into a portable state, allowing them to be repaired and have their lawsets visible.
```

> Rule Lawyering as an AI or Borg with lawsets or custom lawsets defined by antagonists is not fun for anyone.
> Rule Lawering as a imprint implanted person is not fun for anyone, especially the antagonist who used it on you.

Reviewer: "Rule lawyering" is a bit of an unusual term that not many (especially new players) might understand. Could we explain this a little better? Also typo :P
```suggestion
Rule lawyering as an AI or Borg with lawsets or custom lawsets defined by antagonists is not fun for anyone.
Rule lawyering as an imprint-implanted person is not fun for anyone, especially the antagonist who used it on you.
```

> When you are cloned, you forget the last 5 minutes of the events leading up to your death.

Reviewer: Would change this to "revived", as this is not exclusive to the cloning process
```suggestion
When you are revived, you forget the last 5 minutes of the events leading up to your death.
```

Find a bug, want to give feedback or a feature suggestion? Go to our Github Page - https://github.com/UristMcStation/UristMcStation/issues and write it up!
The best antagonist rounds usually tend to include most players in someway.
Losing is !FUN!, each round lost is a lesson to learn and apply for the next round.

> Changelings can take peoples voices, appearance and memories.

Reviewer: ```suggestion
Changelings can assume people's voices, appearance and memories.
```

> Bluespace Revenants will slowly distort overtime without creating runes and feeding on blood.

Reviewer: One for @scrdest , but I _believe_ you simply need to quell whatever 'hunger' you're given. It's not necessarily always runes/blood?
```suggestion
Bluespace Revenants will slowly distort over time without creating runes and feeding on blood.
```

> Fire Extingushers can be used to wet floors to slip people with.

Reviewer: ```suggestion
Fire extinguishers can be used to wet floors to slip people with.
```

You can remove fingerprints on objects by using a rag on them.
You may leave partial fingerprints with specific types of gloves.

> You may leave fibers of identifiable clothing on any object you interact with, swapping to less identifable clothing is always wise before doing something bad.

Reviewer: ```suggestion
You may leave fibers of identifiable clothing on any object you interact with; swapping to less identifiable clothing is always wise before doing something nefarious.
```

> Flashers can repeatidly stun cyborgs and disable their movement and tools.

Reviewer: ```suggestion
Flashers can repeatedly stun cyborgs and disable their movement and tools.
```

Emagging a cyborg only unlocks their access panel, and does not upload custom laws.
You can redirect package-wrapped items using the delivery tagger and deliver them to a specific area.
Items can be thrown down Z-levels to unsuspecting people below.
Screwdrivers allow you to change the timers on a grenade to detonate at different times.

> Insulated Gloves can be redyed to another colour using a crayon in the washing machine.

Reviewer: ```suggestion
Insulated gloves can be redyed to another colour using a crayon in the washing machine.
```

> Budget Insulated Gloves will either reduce or multiply the amount of electrical shock damage you get.

Reviewer: ```suggestion
Budget insulated gloves will either reduce or multiply the amount of electrical shock damage you get.
```

> Leaving the Emitter on in the Engine Room is a unwise decision.

Reviewer: A little pedantic, but it's not _always_ an unwise decision; some setups can take constant emitter blasts
```suggestion
Leaving the Emitter on in the Engine Room is usually an unwise decision.
```

> Emagging the Fax Machine lets you send messages to syndicate contacts.

Reviewer: ```suggestion
Emagging a fax machine lets you send messages to syndicate contacts.
```

Praying to the gods may grant you all sorts of boons, or just a cookie.
The best judgements are always made on the decision of a magic 8-ball.

> It is good practice to remove a victim's headset and turn-off their suit sensors while they are still alive, to avoid medbay or security noticing.

Reviewer: ```suggestion
It is good practice to remove a victim's headset and turn off their suit sensors while they are still alive, to avoid medbay or security noticing.
```

> You can disguise yourself to look like a cyborg by wearing a cardboard helmet and cardboard suit and can control what sprite dependant on your backpack.

Reviewer: ```suggestion
You can disguise yourself to look like a cyborg by wearing a cardboard helmet and cardboard suit; you can control what sprite is used depending on your backpack.
```

> Gunshots and cleaned blood will leave residue which can be found using the Detective's luminol sprayer.

Reviewer: ```suggestion
Gunshots and cleaned blood will leave a residue which can be found using the Detective's luminol sprayer.
```

> You can put your PDA in your ID slot and insert you ID into it in order to save space.

Reviewer: ```suggestion
You can put your PDA in your ID slot and insert your ID into it to save space.
```

> You can delete emails sent to you on your PDA, but they will still be visible on the PDA Messanger Server in Engineering.

Reviewer: ```suggestion
You can delete emails sent to you on your PDA, but they will still be visible on the PDA Messenger Server in Telecommunications.
```

> The camera is the AI's eyes, if the AI is rogue, it is wise to disable the camera nearby to prevent them for seeing you.

Reviewer: ```suggestion
The camera is the AI's eyes, if the AI is rogue, it is wise to disable any cameras nearby to prevent them from seeing you.
```

You can replace the pen in your PDA with other pens, such as para-pens.

> You can replace walls with false-walls, these walls however will not carry over any paint applied above it.

Reviewer: ```suggestion
You can replace walls with false walls, these walls however will not carry over any paint applied above it.
```

You can fill bottles with welding fuel and stuff them with a rag to create a molotov.
The Bluespace Drive is destructible and can create horrible anomalies if destroyed when the ship is jumping.

> You can respawn back into the round after 10 minutes, but remember to spawn as a different character with the memories of what has occured already in the round.

Reviewer: Some odd wording here. You _can_ join as a character with memories that have happened in the round, so long as they cryod and weren't removed. 
```suggestion
You can respawn back into the round after 10 minutes, but remember to spawn as a different character which has not already died in the round. You must not use the memories of your dead characters to your advantage, doing so will result in staff action
```

> Torpedo Warheads can be re-wired into a portable IED with a time-delay, make sure you find the detonator wire before disabling the safeties.

Reviewer: ```suggestion
Torpedo Warheads can be re-wired into a portable IED with a time delay; make sure you find the detonator wire before disabling the safeties.
```

Always tuck the nuclear authentication into bed, sleep tight disky.
You can use an E-Sword to saw through secured suitcases.
You can use a multi-tool to reset secure devices if you need to access them without the code.
You will always spawn with an oxygen mask and tank, use this in emergencies with low air.
The cryo-chambers will always contain a crowbar in the lockers to allow you to escape if the doors are unpowered.
You will always spawn with an inaprovaline auto-injector and gauze. Use this to keep yourself awake and staunch bleeding if attacked until you go to medbay.
You can escape welded lockers by resisting inside of them.
You can add metal into soda-can IEDs to increase the amount of shrapnel created when detonated.

> The best hiding spaces for stolen or illict items can often be in plain sight: O2 lockers, disposals, toolboxes and closets can all serve as great places to store items.

Reviewer: ```suggestion
The best hiding spaces for stolen or illicit items can often be in plain sight: O2 lockers, disposals, toolboxes and closets can all serve as great places to store items.
```

Smugglers Satchels can be hidden underneath a floor tile, making a great small stash for supplies.
The Nuclear Authentication disk can be tracked with the pinpointer device, making stealing it incredibly risky.

> Smart weaponry can be location tracked using the use-of-force computer software, always be careful when stealing smart weapons.

Reviewer: ```suggestion
Smart weaponry can be location-tracked using the use-of-force computer software, always be careful when stealing smart weapons.
```

> Taking a GPS with you when going on Away Missions will increase the likelyhood of being found if deceased.

Reviewer: ```suggestion
Taking a GPS with you when going on away missions will increase the likelihood of being found if deceased.
```

> Going into space while an EVA Ban is in effect is a good way to go missing for the rest of the round.

Reviewer: ```suggestion
Going into space while an EVA ban is in effect is a good way to go missing for the rest of the round.
```

> The Trajan, Antonine and Hadrian can all independantly explore sectors without the ICS Nerva, just ensure to bring a canister for thrust fuel.

Reviewer: ```suggestion
The Trajan, Antonine and Hadrian can all independently explore sectors without the ICS Nerva, just ensure to bring a canister for thrust fuel.
```

> There is a emergency entrance somewhat far from the Permanent Brig - if prepared well enough, you could just reach it.

Reviewer: ```suggestion
There is an emergency entrance somewhat far from the Permanent Brig - if prepared well enough, you could just reach it.
```

> It is good to plan to get caught, especially if committing a grand-crime, as this gives you time to stash escape items.

Reviewer: ```suggestion
It is good to plan to get caught, especially if committing a grand crime, as this gives you time to stash escape items.
```

The Singularity and Supermatter should never touch.
Dionae are unable to commit violent acts.

> Unathi are negatively affected by consuming food or drink with sugar.

Reviewer: ```suggestion
Unathi are negatively affected by consuming food or drinks that contain sugar.
```

Lifts are great at going up and down Z-levels, but it's a good idea not to put anything below them.
Different gas mixes will affect how the Supermatter generates heat and cools, it's always good to ask the Chief Engineer how it works.
Hacking the access wire on a vending machine will allow you to purchase items without the needed access level.

> Chameleon projectors are most effective when projecting a small or tiny item, such as a ciggarette butt, try to match the environment you are in.

Reviewer: ```suggestion
Chameleon projectors are most effective when projecting a small or tiny item, such as a cigarette butt, try to match the environment you are in.
```

Glass shards will cause pain and knock you down if you aren't wearing shoes.
Certain gloves won't fit species without being manually adjusted with wirecutters.
Use the ship's holomaps if you want to get your bearings.

> The AI cannot AI if you take it's power...

Reviewer: ```suggestion
The AI cannot AI if you take its power...
```

The Clown's Noisemaker can make a variety of discreet or annoying sounds, use this to distract someone, make a panic or just have fun!

> You can disable lobby music, ship hum ambience and admin MIDIs using the preference tab.

Reviewer: ```suggestion
You can disable lobby music, ship hum ambience, and admin MIDIs from within the preference tab.
```

> Using your PDA infront of someone will make a audible sound and emote, so it's unwise to use it in a dangerous situation.

Reviewer: ```suggestion
Using your PDA in front of someone will make an audible sound and emote, so it's unwise to use it in a dangerous situation.
```

Traitors will spawn with codewords, you can use these words with potential collaborators to identify if they are syndicate agents.

> As a Team Antagonist, always work together with your fellow antagonists, selling your team out or stating who they are over comms immediatley is against the rules.

Reviewer: ```suggestion
As a team antagonist, always work together with your fellow antagonists; selling your team out or stating who they are over comms immediately is against the rules.
```

> Traitors do not have to mutally work together and could easily backstab on another.

Reviewer: ```suggestion
Traitors do not have to mutually work together and could easily backstab one another.
```

> Changelings can spawn as jobs that might otherwise not be spawnable as traitor antagonist, such as Detectives, Second Officers, etc.

Reviewer: ```suggestion
Changelings can spawn as jobs that might otherwise not be spawn-able as a Traitor antagonist; such as Detectives, Second Officers, etc.
```

Floor Tiles can be sharpened using a wirecutter to create a more deadly thrown projectile.

> Spacesuits have injection ports to administer medicine safely while in low-pressure.

Reviewer: ```suggestion
Spacesuits have injection ports to administer medicine safely while in low pressure environments. These however take extra time to use.
```
